src/hotspot/share/nmt/vmatree.hpp

```diff
@@ -119,7 +121,19 @@ class VMATree {
     }
 
     NativeCallStackStorage::StackIndex stack() const {
-     return sidx;
+      return sidx;
+    }
+
+    NativeCallStackStorage::StackIndex second_stack() const {
+      return second_idx;
```

Comment:
Rename accessors into `reserved_stack` and `committed_stack` and add `bool has_committed_stack()`?

Reply:
Done.